video/out/vulkan/context.c

```diff
@@ -189,17 +189,30 @@ pl_vulkan mppl_create_vulkan(struct vulkan_opts *opts,
 #ifdef VK_EXT_SHADER_OBJECT_EXTENSION_NAME
         VK_EXT_SHADER_OBJECT_EXTENSION_NAME,
 #endif
+        VK_KHR_COOPERATIVE_MATRIX_EXTENSION_NAME,
         VK_KHR_VIDEO_DECODE_QUEUE_EXTENSION_NAME,
         VK_KHR_VIDEO_DECODE_H264_EXTENSION_NAME,
         VK_KHR_VIDEO_DECODE_H265_EXTENSION_NAME,
         VK_KHR_VIDEO_QUEUE_EXTENSION_NAME,
+        VK_KHR_VIDEO_MAINTENANCE_1_EXTENSION_NAME,
+        VK_KHR_SHADER_RELAXED_EXTENDED_INSTRUCTION_EXTENSION_NAME,
```

Comment:
Make sure those are available in our required header version `1.3.238`

Reply:
I will double check - but the CI builds are not enough to confirm this?

Reply:
Unfortunately not, CI runs doesn't run on old enough systems.